resources/crds/ripsaw_v1alpha1_fio_distributed_cr.yaml

```diff
@@ -1,9 +1,14 @@
 apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
 kind: Benchmark
 metadata:
-  name: example-benchmark
+  name: fio-benchmark
   namespace: my-ripsaw
 spec:
+#  elasticsearch:
+#    server: <es_server>
+#    port: 9200
+#    index: fio
```

Comment:
I don't like that we provide a way to change the index. I prefer we explictily state we will use `ripsaw-fio-[results|logs]`. This will help with templating..  Just my .02

Reply:
I'll adjust this. The way I've done it in my latest pgbench commit is to default it in the ripsaw role rather than hard coding "ripsaw-pgbench" in snafu, which technically shouldn't be dependent on ripsaw and so should have index name flexibility. This still allows for an undocumented override of the index name via the CR, but it removes it from being explicitly available as a configurable option.

Reply:
i am also ok with prefixing with snafu-<workload>-<type>